tests/ai_safety/trustyai_service/service/test_trustyai_service.py

```
@@ -21,6 +25,20 @@
     validate_trustyai_service_images,
 )
+from utilities.constants import TRUSTYAI_SERVICE_NAME
+
+logger = structlog.get_logger(__name__)
+
+
+@retry(wait_timeout=60, sleep=5)
+def _wait_for_route_ready(trustyai_service: TrustyAIService, token: str) -> bool:
+    route = trustyai_service.external_route
+    url = f"https://{route}/q/health"
+    response = requests.get(url, headers={"Authorization": f"Bearer {token}"}, verify=False, timeout=10)
```

Ruff (0.15.15)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)